storage/innobase/include/trx0trx.h

```diff
@@ -786,7 +786,7 @@ that modified it is running. */
 typedef std::set<
 	dict_table_t*,
 	std::less<dict_table_t*>,
-	ut_allocator<dict_table_t*> >	trx_mod_tables_t;
+	ut_allocator<std::pair<dict_table_t* const, undo_no_t> > >	trx_mod_tables_t;
```

Comment:
This one looks like it belongs to 10.3, not 10.2. I hope my commit 0c6ff122fa080f80c461f77585176c25c284e33d fixed this.

Reply:
Should indeed, seems correct to me.

Reply:
Just pulled it I did not notice your commit lacked the const modifier e.g.
std::pair<dict_table_t* const, undo_no_t> thus still fails :-(

@vuvova what s your opinion ?

Reply:
current 10.5 definition `ut_allocator<std::pair<dict_table_t* const, trx_mod_table_time_t> > >` so it has const now.